models/users.js

```diff
@@ -215,6 +215,35 @@ const fetchUserImage = async (users) => {
   return images;
 };
 
+/**
+ * Add default status
+ * @returns {Promise<string[]>}
+ */
+const addDefaultStatus = async () => {
+  try {
+    const usersSnapshot = await userModel.get();
+    const users = [];
+    const addedDefaultStatusUsers = [];
```

Comment:
IMO migratedUsers or updatedUsers would be a better variable name

Reply:
`updatedUsers` will be better. As from route, it will be clear what is updated for these users.